src/wepy/runners/openmm.py

```diff
@@ -1166,6 +1166,8 @@ def gen_sim_state(positions, system, integrator,
 
     integrator : openmm.Integrator object
 
+    box_vectors : list of Vec3 Quantities
```

Comment:
To be consistent with the rest of the API this argument should be a numpy array.